src/Desktop.Analyzers/UnitTests/MarkAllNonSerializableFieldsTests.cs

```diff
@@ -45,16 +50,122 @@ Imports System
                 <Serializable>
                 Public Class CA2235WithOnlyPrimitiveFields 
 
-                    Public s1 As String;
-                    Friend s2 As String;
-                    Private s3 As String;
-                    Public i1 As Integer;
-                    Friend i2 As Integer;
-                    Private i3 As Integer;
+                    Public s1 As String
+                    Friend s2 As String
+                    Private s3 As String
+
+                    Public i1 As Integer
+                    Friend i2 As Integer
+                    Private i3 As Integer
+
+                    Public b1 As Boolean
+                    Friend b2 As Boolean
+                    Private b3 As Boolean
                 End Class");
         }
 
         [Fact]
+        public void CA2235WithConstPrimitiveFields()
+        {
+            VerifyCSharp(@"
+                using System;
+    
+                [Serializable]
+                public class CA2235WithConstPrimitiveFields
+                {
+                    public const int i1 = 42;
+                    internal const int i2 = 54;
+                    private const int i3 = 96;
+                }");
+
+            VerifyBasic(@"
+                Imports System
+
+                <Serializable>
+                Public Class CA2235WithConstPrimitiveFields 
+
+                    Public Const i1 As Integer = 42
+                    Friend Const i2 As Integer = 54
+                    Private Const i3 As Integer = 96
+                End Class");
+        }
+
+        [Fact]
+        public void CA2235WithPrimitiveGetOnlyProperties()
+        {
```

Comment:
Add tests for `get`-only properties. @RalfKoban had reported that this didn't work.